src/app/pages/HomePage/index.test.tsx

```diff
@@ -117,6 +121,37 @@ describe('Home Page', () => {
     });
   });
 
+  describe('Lazy Loading', () => {
+    const { BANNER } = VISUAL_STYLE;
+    const { NORMAL } = VISUAL_PROMINENCE;
+    const messageBannerImages = kyrgyzHomePageData.curations
+      .filter(
+        ({ visualStyle, visualProminence }) =>
+          visualStyle === BANNER && visualProminence === NORMAL,
+      )
+      .flatMap(curation => {
+        return curation.summaries?.map(summary => summary.imageUrl);
+      });
+
+    it('Only the first image and message banner on the homepage are not lazy loaded, but all others are', () => {
+      render(<HomePage pageData={homePageData} />, {
+        service: 'kyrgyz',
+      });
+
+      const imageList = document.querySelectorAll('img');
+      imageList.forEach((image, index) => {
+        if (index === 0) {
+          expect(image.getAttribute('loading')).toBeNull();
+        } else {
+          const src = image.getAttribute('src') || '';
+          if (messageBannerImages.includes(src)) {
```

Comment:
Should this logic be inverted? 
```suggestion
          if (!messageBannerImages.includes(src)) {
```